### PR TITLE
Created  Profile Screen User Interface

### PR DESCRIPTION
**Pull request by adarsh-technocrat**
**#128**

fixes #

**Describe the changes you've made**

 Created Complete User Interface for profile Screen,

**`Screen Shots`**
<p align ="left">
  
 

</p>

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

Cc/ @Abhishek-kumar09 , @KeenWarrior

### CODE REVIEW

File: src/pages/ProfilePage/ProfileComponent/HeaderComponent.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  profiletitle: {
+    padding: `${theme.spacing(2)}px`,
+    "& .MuiTypography-root": {
+      fontSize: "1.3rem",
+      fontWeight: 400,
+    },
+  },
+  Avatar: {
```
Comment (Abhishek-kumar09) [inline src/pages/ProfilePage/ProfileComponent/HeaderComponent.js:12]: `Avatar` to `avatar`.

File: src/pages/ProfilePage/ProfileComponent/HeaderComponent.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      fontWeight: 400,
+      marginBottom: `${theme.spacing(0.5)}px`,
+    },
+  },
+  Editbutton: {
+    marginTop: "10px",
+    background: "#fff",
+    textTransform: "none",
+    borderRadius: "30px",
+    border: "1px solid black",
```
Comment (Abhishek-kumar09) [inline src/pages/ProfilePage/ProfileComponent/HeaderComponent.js:43]: Make border 2px or 3px according to the design you have shared

File: src/pages/ProfilePage/ProfileComponent/HeaderComponent.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    padding: "0 5px 0 5px",
```
Comment (Abhishek-kumar09) [inline src/pages/ProfilePage/ProfileComponent/HeaderComponent.js:44]: Add left and right internal padding as like you have did in design.
Add values in multiples of 8.

File: src/pages/ProfilePage/ProfileComponent/ListComponent.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  ListItemText,
+  Typography,
+  Switch as MuiSwitch,
+  makeStyles,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "& .MuiTypography-h1": {
+      marginLeft: `${theme.spacing(2)}px`,
```
Comment (Abhishek-kumar09) [inline src/pages/ProfilePage/ProfileComponent/ListComponent.js:16]: you can do: `marginLeft: theme.spacing(2),`
and that will also work

File: src/pages/ProfilePage/ProfileComponent/ListComponent.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+      fontSize: "1rem",
+      marginBottom: "5px",
+      color: "#0D0F44",
+      fontWeight: 600,
+    },
+    "& .MuiTypography-body2": {
+      fontSize: "0.8rem",
+      color: "#42607D",
+      fontWeight: 500,
+    },
```
Comment (Abhishek-kumar09) [inline src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: I see you have worked hard with providing the exact styles to different typography variant.

Here is the typography configuration: https://github.com/codeforcauseorg/edu-client/blob/main/src/theme/typography.js

If you think that these sizes and weights are not appropriate and is not aligning to use cases then you can create  a new PR that changes these styles.

We don't want developers to do the repetitive things again and that's why we have typography so feel free to change it the most common cases.

I guess the font-sizes mentioned are too small, you can change those it is one of the required things too.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Some items

File: src/pages/ProfilePage/ProfileComponent/HeaderComponent.js (hunk 2 above)
Comment (adarsh-technocrat) [reply src/pages/ProfilePage/ProfileComponent/HeaderComponent.js:43]: Sure @Abhishek-kumar09!

File: src/pages/ProfilePage/ProfileComponent/ListComponent.js (hunk 5 above)
Comment (adarsh-technocrat) [reply src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: Sure @Abhishek-kumar09 I'll create PR for that :)

Comment (adarsh-technocrat) [reply src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: > I guess the font-sizes mentioned are too small, you can change those it is one of the required things too.


is font-size mentioned in the profile page user interface or in the [typography](https://github.com/codeforcauseorg/edu-client/blob/main/src/theme/typography.js) configuration is small?

Comment (Abhishek-kumar09) [reply src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: I was talking about typogaphy, if it looks way to small than the actual used ones.

Comment (adarsh-technocrat) [reply src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: Sure @Abhishek-kumar09  got it :)

Comment (adarsh-technocrat) [reply src/pages/ProfilePage/ProfileComponent/ListComponent.js:36]: Hello, @Abhishek-kumar09  as @faisaljamil25  is working on the typography I things I can go for removing some extra styling :)

**Pull request merged**